samples/plugins/postgres_odbc/connectionBuilder.js

```diff
@@ -11,7 +11,12 @@
     params["LFCONVERSION"] = "0";
     params["UseDeclareFetch"] = "1";
     params["Fetch"] = "2048";
-
+  
+    if (attr[connectionHelper.attributeSSLMode] == "require")
+    {        
+        params["sslmode"] = "require";
+    }
+  
     var odbcConnectStringExtrasMap = {};
     const attributeODBCConnectStringExtras = "odbc-connect-string-extras";
     if (attributeODBCConnectStringExtras in attr) 
```

Comment:
Isn't this a 2020.1 feature?

Reply:
This is moving master into dev, not dev into master. This will not affect master.

Reply:
Except the resolve commit merged dev into master...? Yep, that's not what I expected to happen. I'll fix it.